.github/workflows/cd.yml

```diff
@@ -20,13 +23,18 @@ jobs:
         run: pip install poetry
       - name: Install dependencies
         run: poetry install
+      - name: Run pylint
+        run: poetry run pylint src
       - name: Run tests
         run: poetry run coverage run -m pytest
```

Comment:
We could add a check that coverage is sufficient after this:

```sh
poetry run coverage report --fail-under=80
```

(80% was recommended by Kalle, though the current DoD says 70%)

Reply:
Good idea!